tests/cpp/test_gather.cpp

```diff
@@ -567,8 +569,8 @@ TEST_F(GatherTest, TakeAlongAxisIntermediateTensorPointwise2) {
   Fusion& fusion = *fusion_ptr.get();
   FusionGuard fg(&fusion);
 
-  EnableOptionsGuard opt_guard;
```

Comment:
This was necessary before but not anymore since the `NVFuserTest` fixture itself has one.